.gitignore

```diff
@@ -1,6 +1,7 @@
 build/
 tmp_build/
 src/build/
+src/lib/config_v2/testdata
```

Comment:
I added this because there is a file in testdata `config_API_valid.yml` which keeps growing for each tests or benchmarks as API is added to that file. That's why I added it to the .gitignore. What should be done in this case?